lenskit/crossfold.py

```diff
@@ -345,3 +350,14 @@ def simple_test_pair(ratings, n_users=1000, n_rates=5, f_rates=None):
     train, test = next(sample_users(ratings, 1, n_users, samp))
 
     return train, test
+
+
+def confirm_unique_index(data):
+    """Confirms dataframe has unique index values, and if not,
+     throws ValueError with helpful log message"""
+
+    if not data.index.is_unique:
+        _logger.error("Index column has duplicate values")
+        _logger.info("If index values do not matter, consider running " +
+                     ".reset_index() on the dataframe before partitioning")
+        raise ValueError('Index column is not uniquely valued')
```

Comment:
Error message should just say "index", not "index column" - Pandas doesn't consider the index to be a column.